src/Components/Card/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./styles.css";
+const Card = ({ item }) => {
+  const { title, price, rating, imageURL } = item;
+  return (
+    <div className="card">
+      <img className="item" src={imageURL} />
```

Comment:
You can also add alt to images

Reply:
yes i will do that in the next pr thank you